web/themes/custom/pineapple/components/_heading.twig

```diff
@@ -3,8 +3,14 @@
   heading_level == 1 ? 'page-title'
 ]) %}
 
-{% if heading_url %}
-<h{{ heading_level }}>{% include "@components/_link.twig" %}</h{{ heading_level }}>
-{% else %}
-  <h{{ heading_level }} {{ heading_attributes }}>{{ heading_text }}</h{{ heading_level }}>
-{% endif %}
+<h{{ heading_level }} {{ heading_attributes }}>
+  {% if heading_url %}
+    {% include "@components/_link.twig" with {
+      link_text: heading_text,
+      link_attributes: link_attributes,
+      link_url: link_url
```

Comment:
The second variable should be heading_url so that it's less confusing in places you're including this component